lib/manager/terraform-version/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as hashicorpVersioning from '../../versioning/hashicorp';
+
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)\\.terraform-version$'],
+  versioning: hashicorpVersioning.id,
+  extractVersion: '^v(?<version>.*)$',
```

Comment:
```suggestion
  extractVersion: '^v?(?<version>.*)$',
```

What about suportion the prefix as optional?

Reply:
Terraform only uses `v` prefixed releases and as this repo is as default hardcoded, it makes from my point of view sense to follow that syntax.    
https://github.com/hashicorp/terraform/tags   

If someone want to overwrite this he can do this anytime.